weave_query/weave_query/ops_domain/table.py

```diff
@@ -856,6 +856,13 @@ def file_table(file: artifact_fs.FilesystemArtifactFile) -> typing.Optional[Tabl
         return Table(_get_table_like_awl_from_file(file).awl)
     except FileNotFoundError as e:
         return None
+    # Prevent a panel crash from stale file handle errors
+    # There are rare stale file handle errors that cause panel crashes as noted:
+    # https://wandb.atlassian.net/browse/WB-22355
+    except OSError as e:
+        import errno
+        if e.errno == errno.ESTALE:
+            return None
```

Comment:
I think we'll want to re-raise the exception if it doesn't fall in this case. Something like:
```suggestion
        if e.errno == errno.ESTALE:
            return None
         else:
            raise e
```

Reply:
oh shoot, good catch, thanks @nicholaspun-wandb !

Reply:
could also omit the `else`

```
if e.errno == errno.ESTALE:
   return None
raise
```